shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -23,8 +23,8 @@
             <description>When enabled, the motors will spin at the ChannelNeutral command when armed with zero throttle.</description>
         </field>
 
-        <field name="MotorInputOutputCurveFit" units="-" type="float" elementnames="A,B" defaultvalue="1,1">
-            <description>Actuator mapping of input in [-1,1] to output on [-1,1], using power equation of type a*x^b.</description>
+        <field name="MotorInputOutputCurveFit" units="-" type="float" elements="1" defaultvalue="1">
```

Comment:
I'm not sure what you mean by more linear in the sense of fitting an exponential, but from our test stand data the fit is better for normal purposes. The data seems to show that the "meat" of the curve, i.e. the center point where the propeller spends most of its time, is a better fit with two parameters. That leads to slightly worse fits at the upper and lower ends (because theoretically the ```A``` term should be identically ```1```), but if we're shooting for maximal precision in standard flight this seems like a good compromise.